### DISCUSSION TITLE
Tolgee REST export method idempotence problem

### DISCUSSION THREAD

**Discussion by dvolkovv** [tolgee/tolgee-platform#2473] [Q&A]
Hello, I am working on an Android project consisting of many modules that have strings files inside them.
I wrote a small gradle plugin that accesses the Tolgee service, exports translations by module path and puts them in the /res/values folder of the module
The problem arises when I need to export a lot of strings from different modules. Tolgee sometimes returns a string file in which the line feed for one or two instances is not escaped, other strings might contain normal newline '\n' character.
This does not always occur and not for all files, which is very strange. After completing the gradle task, it can return correctly encoded strings, and everything is fine.
I need the files not to differ when exporting, because I also configured a gradle task that compares the exported strings from Tolgee and those in the module, and crashes if they do not match. This helps me keep up-to-date translations in Tolgee and rely on the service rather than local strings.
I am using self-hosted Tolgee version 3.73.0
<img width="1802" alt="Screenshot 2024-09-19 152615" src="https://github.com/user-attachments/assets/d13873e3-0d40-446e-a96a-67f9b32b5c52">

**Comment by JanCizmar** [comment]
Hey! Thanks for reaching out! 🤩 That's looks weird. Can you please create an bug issue in this repository with exact steps to reproduce it?
